requirements/base.txt

```diff
@@ -16,6 +16,7 @@ serverlessrepo==0.1.10
 aws_lambda_builders==1.29.0
 tomlkit==0.11.7
 watchdog==2.1.2
+rich~=13.3.3
```

Comment:
Any python version requirement that we need to be careful about?

Reply:
rich is supported for 3.7+ which maps to AWS SAM CLI.